docs/starters.yml

```diff
@@ -3071,3 +3071,18 @@
     - Switch the dark mode according to the system theme
     - Scss
     - Pagination
+- url: https://anubhavsrivastava.github.io/gatsby-starter-resume
+  repo: https://github.com/anubhavsrivastava/gatsby-starter-resume
+  description: Single page starter based on the Resume site template by startbootstrap for resume/portfolio page
+  tags:
+    - startbootstrap
```

Comment:
We currently have no tag "startbootstrap" and it doesn't seem really impactful to add.
```suggestion
```